recipes/taskgroup/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set version = "0.0.0a4" %}
```

Comment:
This one falls under https://github.com/conda-forge/cfep/blob/main/cfep-05.md

Reply:
Hm, [no stable release](https://pypi.org/project/taskgroup/#history) exists, and this is [depended upon](https://github.com/conda-forge/hypercorn-feedstock/pull/44) by an existing feedstock.

Reply:
You can place this one is a dev channel and still use it there by adding an extra channel in the conda-forge yaml file. However, that is too cumbersome and, IMO, people should just stop with the dev releases that should not have been a dev release to start with. PyPI set a bad precedent with that "pre" policy IMO.

Reply:
I heartily agree, and really `black` did the worst damage, leaving a lot of people with `--pre` on in various setups. Indeed, having a second `dev` registry, and the _ability_ to specify it per dep would have likely have been more useful, as quietly allowing a `==` pin a pre-release package feels super bad.

Reply:
And black did not need that BTW! It was the only pre release that could be installed as a stable one. Go figure why...